### DISCUSSION TITLE
Lower the volume when talking to Siri?

### DISCUSSION THREAD

**Discussion by leo150** [mikebrady/shairport-sync#1754] [Q&A]
When I play a music on HomePod and ask Siri anything, it lowers the volume. Is there any way to do the same for shairport speaker?

Another question is about the device itself, my speaker always shows "No Response". I think it was working fine before updating to iOS 17. Should I update anything? I'm on `4.3.2-1-g90c03bf0-AirPlay2-smi10-OpenSSL-Avahi-ALSA-soxr-metadata-sysconfdir:/etc`
<details>
  <summary>Screenshot</summary>
  

![IMG_6105](https://github.com/mikebrady/shairport-sync/assets/3484524/0f02e64c-ba97-46bb-9047-49286a9ed036)

</details>

**Comment by mikebrady** [comment]
Thanks for the questions.

1. That lowering of the volume is called "ducking", I believe. The Shairport Sync device is totally dependent on the source device (e.g. the iPhone) for its behaviour, and unfortunately, that behaviour doesn't include ducking.

2. This "No Response" is a new phenomenon, and has been the subject of a few [comments](https://github.com/mikebrady/shairport-sync/issues/1731#issuecomment-1772386876). It seems harmless, but is a bit annoying.
